### PR TITLE
Document inotify config for kind deployments

### LINKED ISSUE #495: [DOC] - Add troubleshooting doc for "too many open files" in kind cluster
### Preliminary Checks

- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/nebari-dev/nebari/discussions

### Summary

Running the local deployment docs https://www.nebari.dev/docs/how-tos/nebari-local did not succeeed.

There were a couple of issues (DNS is still unresolved, but I'll post in a separate issue)

### Steps to Resolve this Issue

`kind` cluster needs a higher ulimit

Resolved by:
```
sudo sysctl fs.inotify.max_user_watches=524288      
sudo sysctl fs.inotify.max_user_instances=512
```

I can add a blurb to https://www.nebari.dev/docs/troubleshooting or https://www.nebari.dev/docs/how-tos/nebari-local if that would be helpful :)

### PR DESCRIPTION
**Pull request by asmacdo**
Fixes #495

## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [X] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### Text styling

- [X] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [X] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [X] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [X] This content adheres to the Nebari style guides.

### CODE REVIEW

File: docs/docs/how-tos/nebari-local.md [hunk 1]
```diff
@@ -159,6 +159,15 @@ security:
         tag: sha-b4a2d1e
 ```
 
+### Increase fs watches
+
+Depending on your host system, you may need to increase the `fs.inotify.max_user_watches` and
+`fs.inotify.max_user_instances kernel parameters` if you see a `too many open files` in the logs of
+a failing pod.
+
```
Comment (kcpevey) [inline docs/docs/how-tos/nebari-local.md:167]: Can you also add the fixer snippet that you listed in the issue? I think that would be beneficial.

```
sudo sysctl fs.inotify.max_user_watches=524288      
sudo sysctl fs.inotify.max_user_instances=512
```

File: docs/docs/how-tos/nebari-local.md [hunk 2]
```diff
@@ -159,6 +159,15 @@ security:
         tag: sha-b4a2d1e
 ```
 
+### Increase fs watches
+
+Depending on your host system, you may need to increase the `fs.inotify.max_user_watches` and
+`fs.inotify.max_user_instances kernel parameters` if you see a `too many open files` in the logs of
```
Comment (kcpevey) [inline docs/docs/how-tos/nebari-local.md:165]:
```suggestion
`fs.inotify.max_user_instances kernel parameters` if you see the error "too many open files" in the logs of
```
The syntax highlighting on the error message being the same as the config variables was confusing to read so I switched it up.

File: docs/docs/how-tos/nebari-local.md [hunk 3]
```diff
@@ -159,6 +159,15 @@ security:
...
+a failing pod.
+
+See the [kind troubleshooting
+docs](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files)
```
Comment (kcpevey) [inline docs/docs/how-tos/nebari-local.md:169]:
```suggestion
docs](https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files) for more information.
```

**Review by kcpevey — CHANGES_REQUESTED** [review]
Hi @asmacdo! Thank you for this contribution! I had a few minor changes to request.

**Comment by marcelovilla** [comment]
@kcpevey can you give this another look so we get it over the finish line? Thanks!

**Review by kcpevey — APPROVED** [review]
@marcelovilla this looks good on my end

**Comment by marcelovilla** [comment]
Thanks @asmacdo for your contribution and @kcpevey for your review!

**Pull request merged**